sdk/iot/core/src/az_iot_core.c

```diff
@@ -25,9 +26,11 @@ AZ_NODISCARD int32_t az_iot_retry_calc_delay(
   _az_PRECONDITION_RANGE(0, max_retry_delay_msec, INT32_MAX - 1);
   _az_PRECONDITION_RANGE(0, random_msec, INT32_MAX - 1);
 
+  az_log_write(AZ_LOG_IOT_RETRY, AZ_SPAN_NULL);
```

Comment:
Do you intentionally want to log null (i.e. empty span)?

Reply:
I would like to log an int similar to ETW but the logging callback doesn't allow that.
I don't want to stack-alloc a span that contains the int values so there's nothing else I can do here.